plenum/test/client/test_client_authn.py

```diff
@@ -22,14 +27,26 @@ def sa(cli):
 
 @pytest.fixture(scope="module")
 def msg():
-    return dict(myMsg="42 (forty-two) is the natural number that succeeds 41 and precedes 43.")
+    return dict(myMsg=msg_str)
 
 
 @pytest.fixture(scope="module")
 def sig(cli, msg):
     return cli.sign(msg)
 
 
+def test_authenticate_raises_correct_exception(testcase=unittest.TestCase()):
+    msg = dict(myMsg=msg_str)
+    simple_signer = SimpleSigner()
+    identifier = simple_signer.identifier
+    signature = simple_signer.sign(msg)
+    verkey = simple_signer.verkey
+    dummyAr = DummyAuthenticator()
+    dummyAr.addIdr(identifier, verkey)
+    testcase.assertRaises(CouldNotAuthenticate, dummyAr.authenticate, msg,identifier, signature)
```

Comment:
We use pytest in our tests.
In pytest you can also check that exception is thrown:
`with pytest.raises(CouldNotAuthenticate):`